src/client/components/ChannelInformation/ChannelInnformation.js

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import './ChannelInformation.style.css';
 
-export const RenderChannelInformation = ({
-  headline = 'Here is channel info',
-}) => {
-  return <h3>{headline}</h3>;
+export const RenderChannelInformation = ({ image, title, members }) => {
+  return (
+    <>
+      <div className="channel-information-header">
+        <img src={image} alt="Room" className="channel-information-img" />
+      </div>
+      <h3 className="channel-information-title">{title}</h3>
+      <div className="channel-information-main">
```

Comment:
You are nesting `div` inside `div` here, but you can also use `ul` and `li` and style them. It's considered far better to use `ul`/`li` instead of `div`/`div` combination.

Reply:
hey @benna100 
would you please approve it to merge so I can send my next pr :-)